pom.xml

```diff
@@ -62,7 +62,7 @@
     <!-- Minimum version of the plugin API in SQ -->
     <sonar.pluginApi.minVersion>9.14.9.375</sonar.pluginApi.minVersion>
 
-    <sonar.analyzerCommons.version>2.7.0.1482</sonar.analyzerCommons.version>
+    <sonar.analyzerCommons.version>2.14.0.3086</sonar.analyzerCommons.version>
```

Comment:
We should be able to drop this change to the POM when analyzer-commons is released. That should also unlock #303

Reply:
OK